exporter/datadogexporter/config.go

```diff
@@ -315,6 +308,20 @@ type LogsConfig struct {
 
 	// DumpPayloads report whether payloads should be dumped when logging level is debug.
 	DumpPayloads bool `mapstructure:"dump_payloads"`
+
+	// LogsDDURL defines the endpoint and port for the logs agent to hit when using a proxy for logs.
+	// The logs are forwarded in TCP, therefore the proxy must be able to handle TCP connections.
+	LogsDDURL string `mapstructure:"logs_dd_url"`
```

Comment:
It seems duplicate of TCPAddrConfig . I don't see a need for it.

Reply:
TCPAddrConfig has a default endpoint value of ["https://http-intake.logs.datadoghq.com"](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/datadogexporter/factory.go#L219), but the logs agent uses a different endpoint [agent-http-intake.logs.datadoghq.com](https://docs.datadoghq.com/logs/log_collection/?tab=host#supported-endpoints).

[logs_dd_url](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml#L907-L912) is optional in the datadog-agent config, so we don't need to set a default value for this.